requirements/nightly.in

```diff
@@ -4,7 +4,7 @@
 dask
 graphviz
 pythreejs
-https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

Comment:
How come this is changed?

Reply:
Good question. All I did is run `tox -e deps`

Reply:
... in the wrong env?

Reply:
I used python 3.10 to lock dependencies. I re did it with 3.9 now which reverted the change here.